guide/interactions/select-menus.md

```diff
@@ -194,30 +211,46 @@ client.on(Events.InteractionCreate, async interaction => {
 	}
 });
 ```
+## Getting options
+
+To get options than user seleted, you can access <DocsLink path="class/StringSelectMenuInteraction?scrollTo=values" /> which is an array of options' values or access a Collection with objects like <DocsLink path="class/RoleSelectMenuInteraction?scrollTo=roles" />, <DocsLink path="class/ChannelSelectMenuInteraction?scrollTo=channels" /> ... 
```

Comment:
```suggestion
To get options that the user seleted, you can access <DocsLink path="class/StringSelectMenuInteraction?scrollTo=values" /> which is an array of option values or access the Collection with objects through <DocsLink path="class/RoleSelectMenuInteraction?scrollTo=roles" />, <DocsLink path="class/ChannelSelectMenuInteraction?scrollTo=channels" />
```
since values is plural we must not use option as plural